### ISSUE TITLE
Peer Review Publication / Подготовка Научной Статьи

### ISSUE THREAD

**Issue by denisbanchenko** [AdvancedScientificResearchProjects/Hyperbolic_Field_BloodPlasma_Study#8]
## 📑 PEER REVIEW PUBLICATION PREPARATION / ПОДГОТОВКА НАУЧНОЙ СТАТЬИ

**Status / Статус:** 🟡 In Production / В Производстве  
**Expected Publication / Ожидаемая Публикация:** 1-2 months (April-May 2026) / 1-2 месяца (апрель-май 2026)  
**Journal / Журнал:** ASRP.science  
**Lead / Руководитель:** Ivan Savelyev / Иван Савельев

## 📅 PUBLICATION TIMELINE / СРОКИ ПУБЛИКАЦИИ

```mermaid
gantt
    title Publication Timeline / Сроки Публикации
    dateFormat  YYYY-MM-DD
    
    section Preparation / Подготовка
    Manuscript Draft / Черновик рукописи      :active,  des1, 2026-03-01, 30d
    Internal Review / Внутренний обзор       :         des2, after des1, 14d
    External Peer Review / Внешнее рецензирование  :         des3, after des2, 30d
    
    section Publication / Публикация
    Final Edits / Финальные правки           :         des4, after des3, 7d
    Publication / Публикация           :         des5, after des4, 7d
```

### Q1: Are LLM prompts available in the repository? / В1: Доступны ли промпты LLM в репозитории?

**A:** Yes, all prompts are stored in `scripts/llm_analysis/prompts.py`

**О:** Да, все промпты хранятся в `scripts/llm_analysis/prompts.py`

**Location / Расположение:**
- Blinded prompts, Unblinded prompts, Comparative prompts / Ослеплённые промпты, раскрытые промпты, сравнительные промпты
- Batch prompts, Multi-tube prompts / Пакетные промпты, мультипробирочные промпты

**Scripts / Скрипты:**
- `run_comparative.py`, `run_multi_tube.py`, `run_batch.py`
- `providers.py`, `imaging.py`

---

### Q2: Why use general-purpose LLMs instead of specialized medical models? / В2: Почему используются LLM общего назначения, а не специализированные медицинские модели?

**A:** We tested BOTH approaches. Here are the results:

**О:** Мы протестировали ОБА подхода. Вот результаты:

| Model / Модель | Type / Тип | Accuracy / Точность | Conclusion / Вывод |
|---------------|-----------|-------------------|-------------------|
| **Gemini 2.5 Flash** | General LLM / Общая LLM | **57.9%** (p=0.027) | ✅ Statistically significant / Статистически значимо |
| **DINOv2 Linear Probe** | CV Model / CV модель | **47.4%** (p=0.15) | 🟡 Suggestive / Предположительно |
| **BiomedCLIP** | Specialized Medical / Спец. медицинская | **36.8%** | ❌ Chance level / Уровень случайности |
| **MedSigLIP** | Specialized Medical / Спец. медицинская | **N/A** | ❌ Out-of-distribution / Вне распределения |
| **SigLIP2 Zero-Shot** | General CV / Общая CV | **26-37%** | ❌ Chance level / Уровень случайности |

**Key Findings / Ключевые Находки:**
1. ✅ Specialized medical models performed at chance level / Специализированные медицинские модели показали уровень случайности
2. ✅ General-purpose LLMs found signal (57.9%, statistically significant) / LLM общего назначения обнаружили сигнал (57.9%, статистически значимо)
3. ✅ Blinding protocol prevents hallucination (Perplexity reversal: 53% → 0%) / Протокол ослепления предотвращает галлюцинации (разворот Perplexity: 53% → 0%)
4. ✅ Results reproduced across different architectures (LLM + CV) / Результаты воспроизведены на разных архитектурах (LLM + CV)

---

### Q3: How do you control for LLM hallucination? / В3: Как вы контролируете галлюцинации LLM?

**A:** Multi-layer protocol:

**О:** Многоуровневый протокол:

1. **Blinding Protocol / Протокол Ослепления**
   - Models see "Sample A / B / C" only / Модели видят только "Образец A / B / C"
   - Don't know which channel is which / Не знают, какой канал какой
   - Perplexity reversal proves blinding works (53% → 0%) / Разворот Perplexity доказывает работоспособность ослепления (53% → 0%)

2. **Triple Run Validation / Тройная Проверка**
   - Each analysis run 3 times on same data / Каждый анализ запускается 3 раза на одних и тех же данных
   - Checks reproducibility / Проверяет воспроизводимость
   - Identifies hallucinations / Выявляет галлюцинации

3. **Dual-Track Analysis / Двухтрековый Анализ**
   - Track 1: LLM analysis (Gemini, GPT, Perplexity) / Трек 1: Анализ LLM (Gemini, GPT, Perplexity)
   - Track 2: CV/ML models (DINOv2, SigLIP2) - mathematically deterministic / Трек 2: CV/ML модели (DINOv2, SigLIP2) - математически детерминированные
   - Compare results between tracks / Сравнение результатов между треками

---

### Q4: Why not fine-tune models on plasma images? / В4: Почему бы не дообучить модели на изображениях плазмы?

**A:** Planned for next phase. Current limitations:

**О:** Запланировано на следующую фазу. Текущие ограничения:

1. **Dataset Size / Размер Датасета**
   - Current: 5 patients (19 triplets) / Текущий: 5 пациентов (19 триплетов)
   - Fine-tuning would overfit on this size / Дообучение переобучится на этом размере
   - Need larger dataset first / Сначала нужен больший датасет

2. **No Existing Plasma Models / Не Существует Моделей для Плазмы**
   - No public models trained on plasma tube photography / Нет публичных моделей, обученных на фотографиях пробирок с плазмой
   - Closest domains: clinical pathology, biomedical literature / Ближайшие домены: клиническая патология, биомедицинская литература
   - Both tested (BiomedCLIP, MedSigLIP) - no advantage / Оба протестированы (BiomedCLIP, MedSigLIP) - без преимущества

3. **Next Steps / Следующие Шаги**
   - Expand donor base to 30 participants (Issue #4) / Расширить базу доноров до 30 участников (Issue #4)
   - Then fine-tune DINOv2 or similar / Затем дообучить DINOv2 или аналогичную
   - Full fine-tuning makes sense with sufficient data / Полное дообучение имеет смысл при достаточных данных

---

### Q5: What is the evidence that effect exists? / В5: Какие есть доказательства существования эффекта?

**A:** Despite all limitations, multiple models found signal:

**О:** Несмотря на все ограничения, несколько моделей обнаружили сигнал:

| Model / Модель | Accuracy / Точность | P-value / P-значение | Status / Статус |
|---------------|-------------------|-------------------|----------------|
| **Gemini 2.5 Flash** | 57.9% | p = 0.027 | ✅ Statistically significant / Статистически значимо |
| **DINOv2 Linear Probe** | 47.4% | p = 0.15 | 🟡 Suggestive / Предположительно |
| **GPT-5 Batch** | 46.7% | - | 🟡 Consistent / Согласованно |
| **Perplexity Batch** | 53.3% | - | 🟡 Consistent / Согласованно |

**Conclusion / Вывод:**
- If no effect existed, all models would show ~33% (chance level) / Если бы эффекта не было, все модели показали бы ~33% (уровень случайности)
- Instead, multiple architectures show 46-58% accuracy / Вместо этого несколько архитектур показывают точность 46-58%
- This is not proof of effect, but reproducible signal across different models / Это не доказательство эффекта, но воспроизводимый сигнал на разных моделях

## 📞 CONTACT INFORMATION / КОНТАКТНАЯ ИНФОРМАЦИЯ

| Contact / Контакт | Email / Электронная почта | Role / Роль |
|------------------|--------------------------|-------------|
| **👨‍💼 BANCHENKO DENIS YURIEVICH / БАНЧЕНКО ДЕНИС ЮРЬЕВИЧ** | denisbanchenko@asrp.tech | **CEO ASRP / Program Director / Директор Программы** |
| **👩‍⚕️ OVSYANNIKOVA VALERIA ALEXANDROVNA / ОВСЯННИКОВА ВАЛЕРИЯ АЛЕКСАНДРОВНА** | valeriaovseannicova@asrp.tech | **CBE (Chief Biomedical Engineer) / Director of Biomedical Research Department / Руководитель Департамента Биомедицинских Исследований** |
| **👨‍💻 KAPUSTIN MYKHAILO MYKHALOVICH / КАПУСТИН МИХАЙЛО МИХАЙЛОВИЧ** | mykhailokapustin@asrp.tech | **CTO (Chief Technology Officer) / Director of IT & AI Department / Директор Департамента Информационных Технологий и ИИ** |
| **🔬 ZMIENKO KYRYL / ЗМИЕНКО КИРИЛЛ** | kyrylzmiienko@asrp.tech | **Chief AI Engineer / Главный ИИ Инженер** |
| **📚 SAVELYEV IVAN / САВЕЛЬЕВ ИВАН** | ivansavelev@asrp.science | **Science Director / Editor-in-Chief ASRP.science / Директор по Науке и Главный Редактор** |

**Last Updated / Последнее обновление:** 25 March 2026 / 25 марта 2026  
**Journal / Журнал:** [ASRP.science](https://asrp.science)  
**Expected Publication / Ожидаемая Публикация:** April-May 2026 / апрель-май 2026
